pihole

```diff
@@ -147,7 +147,7 @@ versionFunc() {
   exec "${PI_HOLE_SCRIPT_DIR}"/version.sh
 }
 
-restartDNS() {
+reloadDNS() {
   local svcOption svc str output status pid icon FTL_PID_FILE
   svcOption="${1:-restart}"
```

Comment:
You need to change the option handling here. `restart is gone`
```suggestion
  svcOption="${1:-reload}"
```

Reply:
Good catch